src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js

```diff
@@ -25,10 +26,15 @@ const Body = BaseForm.extend(Object.assign(
         messagesObjs.value.forEach(messagesObj => {
           const msg = messagesObj.message;
           if (messagesObj?.class === 'ERROR') {
-            this.add(createCallout({
+            let options = {
               content: msg,
               type: 'error',
-            }), '.o-form-error-container');
+            };
+            if (messagesObj?.i18n?.key === OV_NMC_FORCE_UPGRAGE_KEY) {
+              // add a title for OV force upgrade
+              options.title = loc('idx.authenticator.app.method.push.force.upgrade.number_challenge.title', 'login');
+            }
+            this.add(createCallout(options), '.o-form-error-container');
```

Comment:
Can you confirm that this error gets read aloud via a screenreader for a11y purposes?

I don't see any reason why it wouldn't, but we should probably start confirming we do not introduce breaking a11y behavior whenever we add new DOM elements to a page.